app/api/auth.py

```diff
@@ -222,7 +226,14 @@ def reset_password_post():
         if user.was_registered_with_order:
             send_email_with_action(user, PASSWORD_RESET_AND_VERIFY, app_name=get_settings()['app_name'], link=link)
         else:
-            send_email_with_action(user, PASSWORD_RESET, app_name=get_settings()['app_name'], link=link)
+            previous_hour = datetime.now(pytz.utc) - timedelta(hours=1)
```

Comment:
@shreyanshdwivedi Try datetime.utcnow

Reply:
@prateekj117 when I used `datetime.utcnow()`, I'm getting datetime without timezone - `2019-06-12 12:54:13.035101` - neither `00:00` nor `05:30`

Reply:
The now() call returns the correct time for local location, while the utcnow() call returns the time in the UTC time zone.

Reply:
The time returned by both are absolutely correct. I just want to include my current timezone

Reply:
Try datetime.utcnow().isoformat()